components/msg/Cargo.toml

```diff
@@ -13,5 +13,5 @@ path = "lib.rs"
 bitflags = "0.7"
 heapsize = "0.4"
 heapsize_derive = "0.1"
-serde = "1.0"
+serde = { version = "1.0.11", features = [ "unstable" ] }
```

Comment:
Make that `1.0.14` or use `1.0.11` in Cargo.lock, better to keep them in sync to avoid surprises later.

Reply:
Fixed.